packages/nextjs/src/config/define-config.test.ts

```diff
@@ -1,27 +1,374 @@
+      });
+
+      after(() => {
+        delete process.env.NEXT_PUBLIC_SITECORE_SITE_NAME;
+      });
+
+      it('should use the value from the config if present', () => {
+        defineConfigModule.defineConfig({
+          ...defaultConfig(),
+          defaultSite: 'skate-park',
+        });
+        const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+        expect(resultConfig.defaultSite).to.equal('skate-park');
+      });
+
+      it('should use the env var if config value not present', () => {
+        defineConfigModule.defineConfig(defaultConfig());
+        const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+        expect(resultConfig.defaultSite).to.equal('next-public-sitecore-site-name');
+      });
+    });
+  });
+
+  describe('config.defaultLanguage', () => {
+    describe('environment variable is not set', () => {
+      it('should default to undefined', () => {
+        defineConfigModule.defineConfig({
+          api: {
+            edge: { contextId: 'contextId' },
+          },
+        });
+        const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+        expect(resultConfig.defaultLanguage).to.be.undefined;
+      });
+
+      it('should use the value from the config', () => {
+        defineConfigModule.defineConfig(defaultConfig());
+        const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+        expect(resultConfig.defaultLanguage).to.equal('en');
+      });
+    });
+
+    describe('environment variable is set', () => {
+      before(() => {
+        process.env.NEXT_PUBLIC_DEFAULT_LANGUAGE = 'de';
+      });
+
+      after(() => {
+        delete process.env.NEXT_PUBLIC_DEFAULT_LANGUAGE;
+      });
+
+      it('should use the value from the config if present', () => {
+        defineConfigModule.defineConfig(defaultConfig());
+        const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+        expect(resultConfig.defaultLanguage).to.equal('en');
+      });
+
+      it('should use the env var if config value not present', () => {
+        defineConfigModule.defineConfig({
+          api: {
+            edge: { contextId: 'contextId' },
+          },
+        });
+        const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+        expect(resultConfig.defaultLanguage).to.equal('de');
+      });
+    });
+  });
+
+  describe('config.multisite', () => {
+    it('should default to undefined', () => {
+      defineConfigModule.defineConfig(defaultConfig());
+      const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+      expect(resultConfig.multisite?.enabled).to.be.undefined;
+    });
+
+    it('should be able to override default value of enabled', () => {
+      defineConfigModule.defineConfig({ ...defaultConfig(), multisite: { enabled: false } });
+      const resultConfig = defineConfigCoreStub.getCalls()[0].args[0];
+      expect(resultConfig.multisite?.enabled).to.be.false;
+    });
```

Comment:
this is related to `enabled` property, right?
let's group it under `describe('enabled')`